app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -239,6 +239,11 @@ export default Controller.extend(bufferedProperty("model"), {
     return Category.findById(categoryId)?.minimumRequiredTags || 0;
   },
 
+  @discourseComputed("model.posts_count")
+  showBottomTopicMap(postsCount) {
+    return postsCount > 10;
```

Comment:
Should we be using the `MIN_POST_COUNT` constant defined in `topic-map-summary`?